pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -51,17 +76,17 @@ func GetBootstrapOpenshiftRoles(openshiftNamespace string) []authorizationapi.Ro
 }
 
 func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
+
+	// four resource can be a single line
+	// up to ten-ish resources per line otherwise
+
 	roles := []authorizationapi.ClusterRole{
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: ClusterAdminRoleName,
 			},
 			Rules: []authorizationapi.PolicyRule{
-				{
-					APIGroups: []string{authorizationapi.APIGroupAll},
-					Verbs:     sets.NewString(authorizationapi.VerbAll),
-					Resources: sets.NewString(authorizationapi.ResourceAll),
-				},
+				authorizationapi.NewRule("*").Groups("*").Resources("*").RuleOrDie(),
 				{
```

Comment:
> authorizationapi.NewRule("*").NonResourceURLs("*").RuleOrDie(),

Not necessary for 95% of the problem and this is a "solve 95% of this before anyone merges another pull touching policy." pull.